src/libs/address/useFormatFullAddress.ts

```diff
@@ -25,6 +25,21 @@ export function formatFullAddress(
   return fullAddress
 }
 
+export function formatFullAddressStartsWithPostalCode(
+  address: FullAddressType,
+  postalCode: FullAddressType,
+  city: FullAddressType
+) {
+  if (!address && !postalCode && !city) return ''
+  if (address && !postalCode && !city) return address
+
+  const firstPart = [postalCode, city].filter(Boolean).join(' ')
+
+  if (firstPart && !address) return firstPart
+
+  return [firstPart, address].join(', ')
```

Comment:
Tu peux remplacer par

```suggestion
  return [firstPart, address].filter(Boolean).join(', ')
```